src/setup_node_env/node_version_validator.test.js

```diff
@@ -68,4 +68,25 @@ describe('NodeVersionValidator', function () {
       done();
     });
   });
+
+  it('should run the script WITHOUT error', function (done) {
```

Comment:
Can you add additional details to the test name to distinguish it from the test above?

Reply:
Done.